app/src/test/java/edu/uc/mille5bm/smartnutrition/ExampleUnitTest.kt

```diff
@@ -14,4 +14,14 @@ class ExampleUnitTest {
     fun addition_isCorrect() {
         assertEquals(4, 2 + 2)
     }
-}
\ No newline at end of file
+    
+    @Test
+    fun confirmExercise(){
+        var workout = Exercise() 
+    }
+    
+    @Test
+    fun confirmFood(){
+    var munchies = Food() 
+    }
```

Comment:
The @Test annotation is a bit of a misnomer here.  These aren't tests... there's no assert.  They are simply instantiating objects.  Thus, this gives an impression that more tests exist than actually do.